src/Host/Client/Impl/DataInspection/IREvaluationResultInfo.cs

```diff
@@ -58,10 +58,11 @@ public static class REvaluationResultInfoExtensions {
             this IREvaluationResultInfo info,
             REvaluationResultProperties properties,
             string repr,
+            bool evaluateActiveBindings,
```

Comment:
I'd make it a new flag in `REvaluationResultProperties` instead, since it's effectively about whether the `Value` property of the result is filled or not.